fly_manager.py

```diff
@@ -154,10 +152,32 @@ def import_secrets(self):
             if returncode != 0:
                 raise Exception("Error importing secrets to Fly")
 
+    def save_app_hostname_to_env(self):
+        process = subprocess.run(
+            ["flyctl", "status", "--app", self.app_name, "--json"],
+            stdout=subprocess.PIPE,
+            text=True,
+        )
+
+        # Parse the output as a JSON object
+        try:
+            info = json.loads(process.stdout)
+            print(info)
+            host_name = info["Hostname"]
+            domain = "https://" + host_name
```

Comment:
you can access domain using Django site framework or env because you don't know server is running on the "http" or "https".